source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,749 @@
+
+Now, let’s analyze the statistical trends using `PERMANOVA`_. Permanova tests the hypothesis that samples within a group are more similar to each other than they are to samples in another group. To put it another way, it tests whether the within-group distances from each group are different from the between group distance. We expect samples that are similar to have smaller distances from each other, so if our hypothesis that one group is different from another is true, we’d expect the within-group distances to be smaller than the between group distance.
+
+Let’s use the command to test whether the donor identity (which we identified as a major separator in PCoA space) is associated with significant differences in weighted and unweighted UniFrac distance.
+
+.. command-block::
+
+   qiime diversity beta-group-significance \
+     --i-distance-matrix core-metrics-results/unweighted_unifrac_distance_matrix.qza \
+     --m-metadata-file metadata.tsv \
+     --m-metadata-column donor \
+     --o-visualization core-metrics-results/unweighted-unifrac-donor-significance.qzv
+
+   qiime diversity beta-group-significance \
+     --i-distance-matrix core-metrics-results/weighted_unifrac_distance_matrix.qza \
+     --m-metadata-file metadata.tsv \
+     --m-metadata-column donor \
+     --o-visualization core-metrics-results/weighted-unifrac-donor-significance.qzv
+
+Let’s also check whether there’s a relationship between cage where a mouse lives and the beta diversity, since this is often an important technical effect to consider. Since we have several cages, we’ll use the ``--p-pairwise`` parameter that will let us check whether there are individual differences between the cages driving the difference. This may be useful, since if we check the metadata, we may find that cage is nested by donor.
+
+.. command-block::
+
+   qiime diversity beta-group-significance \
+     --i-distance-matrix core-metrics-results/unweighted_unifrac_distance_matrix.qza \
+     --m-metadata-file metadata.tsv \
+     --m-metadata-column cage_id \
+     --o-visualization core-metrics-results/unweighted-unifrac-cage-significance.qzv \
+     --p-pairwise
+
+   qiime diversity beta-group-significance \
+     --i-distance-matrix core-metrics-results/weighted_unifrac_distance_matrix.qza \
+     --m-metadata-file metadata.tsv \
+     --m-metadata-column cage_id \
+     --o-visualization core-metrics-results/weighted-unifrac-cage-significance.qzv \
+     --p-pairwise
+
+We can use the adonis function to look at a multivariate model. Let’s look at the intersection between donor and genotype.
+
+.. command-block::
+
+   qiime diversity adonis \
+     --i-distance-matrix core-metrics-results/unweighted_unifrac_distance_matrix.qza \
+     --m-metadata-file metadata.tsv \
+     --o-visualization core-metrics-results/unweighted_adonis.qzv \
+     --p-formula genotype+donor
+
+.. do we also want permadisp here?
```

Comment:
I will defer to you and @nbokulich here.

Reply:
I like it from a conceptual perspective and practice running the command, but Im not sure it adds that much. @nbokulich?

Reply:
I think a brief mention/example would be nice. We do not talk about permdisp in other tutorials; unless if we write an in-depth tutorial on diversity analyses at some point (not a bad idea!), it could be good to start using permdisp in our workshop tutorial(s).

Reply:
Okay... I added permdisp for cage. Its still not a great example (I sort of wish I had overlap in this tutorial), but that may be for a later, more indepth option. Id be interested in contributing to that sort of thing at a later date.